e2e/tests/test_e2e.py

```diff
@@ -292,7 +292,7 @@ def count_pods_with_label(self, labels, namespace='default'):
     def wait_for_master_failover(self, expected_master_nodes, namespace='default'):
         pod_phase = 'Failing over'
         new_master_node = ''
-        labels = 'spilo-role=master,version=acid-minimal-cluster'
+        labels = 'spilo-role=master,cluster-name=acid-minimal-cluster'
```

Comment:
I dont like that this is hard coded. We should have the end to end test covering that the label can change to make sure config variable is picked up.

Reply:
Yeah, but there are even more selectors hard coded here: spilo-role, name etc.
But true, I could add a test on changing such labels